### PR TITLE
Solution_704

### LINKED ISSUE #935: Leetcode Question Number 704
Leetcode Question Number 704 solution needs to be added in the repo . 
 The link for the question is [this](https://leetcode.com/problems/binary-search).
 Accepted Languages C++ / JAVA . 
 You Only need to add the  class Solution in the file. 
 Make Sure you have read the contributions guidelines before creating an PR.

### PR DESCRIPTION
**Pull request by Atif0604**
Fixes #935 

My code sample is in : 
- [X] CPP
- [ ] Java 

Screenshot of working code sample  : 

image

## Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] My code sample is formatted and works correctly 

[Contributor Guide]: https://github.com/amanv8060/Leetcode-Questions/blob/main/CONTRIBUTING.md

### CODE REVIEW

**Comment by Atif0604** [comment]
Hey @amanv8060
please review it

File: solutions/704.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+class Solution {
+ public:
+  int search(vector<int>& nums, int target) {
+    int s = 0;
+    int e = arr.size() - 1;
```
Comment (amanv8060) [inline solutions/704.cpp:8]: nums.size()

**Comment by amanv8060** [comment]
This pr has been reverted .

**Comment by Atif0604** [comment]
Hey @amanv8060 do i need to make PR again?

**Pull request merged**